typings/openverse/index.d.ts

```diff
@@ -3,8 +3,8 @@ declare module '*.svg' {
   export default SVG
 }
 
-declare module '*.svg!inline' {
-  const SVG: string
+declare module '*.svg?inline' {
+  const SVG: unknown
```

Comment:
It's nice that at least this removes the warning on files using these imports :)